extensions/wikia/AdEngine/js/tracking/adInfoTracker.js

```diff
@@ -45,6 +46,7 @@ define('ext.wikia.adEngine.tracking.adInfoTracker',  [
 			'pv': pageParams.pv || '',
 			'pv_unique_id': win.pvUID,
 			'browser': [ browserDetect.getOS(), browserDetect.getBrowser() ].join(' '),
+			'device': megaAdUnitBuilder.getDevice(pageParams),
```

Comment:
I believe we don't want to have results like `unknown-specialpage` :P. Can we avoid using megaAdUnitBuilder for that? What about extracting separate module for device and use it here and in megaAdUnitBuilder?

Reply:
Done!